WooCommerce/Classes/ViewRelated/Orders/OrderListViewController.swift

```diff
@@ -154,12 +158,15 @@ final class OrderListViewController: UIViewController, GhostableViewController {
         registerTableViewHeadersAndCells()
         configureTableView()
 
-        configureViewModel()
-        configureSyncingCoordinator()
-
         if ServiceLocator.featureFlagService.isFeatureFlagEnabled(.IPPInAppFeedbackBanner) {
-            viewModel.displayIPPFeedbackBannerIfEligible()
+            guard let survey = viewModel.displayIPPFeedbackBannerIfEligible() else {
+                return
+            }
+            IPPsurveyVariation = survey
```

Comment:
Do you actually need this to be in a guard? Could you just always set the property to be whatever the viewModel returns? Logic to ignore the nil case here seems like it may cause bugs in future. 

I'd expect setting a `nil` survey to hide the banner, but it seems like maybe that would be ignored and we'd continue showing the previous banner, if there ever had been one.

Reply:
Oh, we don't! Removed on 1dc6c4691149595603e6f898b07e0e558cfe5cc8